packages/apfelxx/download.sh

```diff
@@ -0,0 +1,3 @@
+URL=$1
+
+git clone $URL
```

Comment:
To improve the stability of the packages, I was trying to download releases, as much as possible.

The `apfel` package at the moment is based on a commit, because there was not a recent enough release including that commit.
If that's the case for `apfelxx`, please just do the same.

Since we only need the code, not the full Git database, just use `wget`. The link for a release you can simply get from its tag, or the associated release page.
E.g., for latest Apfel++ you can find the release here:
https://github.com/vbertone/apfelxx/releases/tag/4.7.0
and the link to the code is:
https://github.com/vbertone/apfelxx/archive/refs/tags/4.7.0.tar.gz